### PR TITLE
Support amp-ad for LOKA Research.inc

### PR DESCRIPTION
**Pull request by yoppi**
LOKA Research.inc have supported amp-ad.
I was wandering if you review this pull-request.

### CODE REVIEW

**Comment by lannka** [comment]
we recently added new API: [renderStart](https://github.com/ampproject/amphtml/blob/master/ads/README.md#methods-available-to-the-ad)
Could you take a look and implement the APIs (renderStart + noContentAvailable)?
It helps AMP to provide a better ad loading experience.
Let me know if you have any questions regarding that.

**Comment by yoppi** [comment]
@lannka Thanks for review.
We'd tried using `renderStart`, `noContentAvailable` API.

**Comment by lannka** [comment]
@yoppi just a heads up: you might want to sync to head to resolve some conflicts before implementing the new APIs.

**Comment by yoppi** [comment]
@lannka Thanks! We'd resolved conflict. Please review again.

File: examples/ads.amp.html [hunk 1]
```diff
@@ -549,6 +549,12 @@ <h2 id="kargo">Kargo</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="loka">LOKA</h2>
+  <amp-ad width="300" height="250"
+    type="loka"
```
Comment (lannka) [inline examples/ads.amp.html]: nit: please follow the indentation

File: examples/ads.amp.html [hunk 2]
```diff
@@ -549,6 +549,12 @@ <h2 id="kargo">Kargo</h2>
...
+    data-unit-params='{"unit":"mvbanner","id":"Oriowd6zgmpXa9Eg"}'>
+  </amp-ad>
```
Comment (lannka) [inline examples/ads.amp.html]: Please add placeholder and fallback as other example does

File: examples/ads.amp.html [hunk 3]
```diff
@@ -549,6 +549,12 @@ <h2 id="kargo">Kargo</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="loka">LOKA</h2>
```
Comment (lannka) [inline examples/ads.amp.html]: Please add a link in the corresponding section at the top of this file.

File: ads/loka.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+const lokaFields = ['unit_params'];
```
Comment (lannka) [inline ads/loka.js]: should be `unitParams`

**Comment by yoppi** [comment]
@lannka Thanks review. 
Could you teach us, how to local developing. 
We tried to lunch localserver with `gulp` and accessed to `http://ads.localhost:8000/examples/ads.amp.html`.  But, amp scripts like `amp-ad-0.1.js` is could not use local one.

**Comment by lannka** [comment]
@yoppi use `/examples/ads.amp.max.html`

**Comment by yoppi** [comment]
@lannka Thank you for follow-up. We fixed code and checked on `ads.amp.max.html`. Please review again.

**Pull request merged**